### PR TITLE
Metis and cutoff fixes

### PR DESCRIPTION
**Pull request by tomasbruna**

### CODE REVIEW

File: modules/chameleon/src/main/java/org/clueminer/chameleon/PairMerger.java [hunk 1]
```diff
@@ -173,8 +173,8 @@ protected void addIntoTree(int clusterIndex1, int clusterIndex2) {
         if (sim > 10) {
             sim = 10;
         }
-        if (sim < 0.01) {
-            sim = 0.01;
+        if (sim < 0.005) {
```
Comment (deric) [inline modules/chameleon/src/main/java/org/clueminer/chameleon/PairMerger.java]: This should really be listed between parameters.

File: modules/clustering-eval/src/main/java/org/clueminer/eval/hclust/FirstJump.java [hunk 2]
```diff
@@ -15,6 +15,11 @@
 
     public static final String name = "First jump cutoff";
 
+    // Values with best average performance found by several experiments
+    // Values 100 and 2 give much better results on specific datasets but not on average
+    private int start = 340;
+    private double factor = 1.91;
```
Comment (deric) [inline modules/clustering-eval/src/main/java/org/clueminer/eval/hclust/FirstJump.java]: These values also seems like parameters to me.

**Comment by deric** [comment]
Thanks, we convert those values to parameters in next PR.

File: modules/clustering-eval/src/main/java/org/clueminer/eval/hclust/FirstJump.java (hunk 2 above)
Comment (tomasbruna) [inline modules/clustering-eval/src/main/java/org/clueminer/eval/hclust/FirstJump.java]: Yes, these are definitely parameters, will convert them them

File: modules/chameleon/src/main/java/org/clueminer/chameleon/PairMerger.java (hunk 1 above)
Comment (tomasbruna) [inline modules/chameleon/src/main/java/org/clueminer/chameleon/PairMerger.java]: I am trying to find a method to set this value dynamically, this is work in progress.

Comment (deric) [inline modules/chameleon/src/main/java/org/clueminer/chameleon/PairMerger.java]: Ok, no problem. Eventually we might want to override the automatic setting.

**Pull request merged**